mailer.js

```diff
@@ -11,6 +12,24 @@ var DEFAULT_LOCALE = 'en'
 var DEFAULT_TIMEZONE = 'Etc/UTC'
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var templateNameToCampaignMap = {
+    'passwordResetRequiredEmail': 'password-reset-required',
+    'passwordChangedEmail': 'password-changed-success',
+    'passwordResetEmail': 'password-reset',
+    'postVerifyEmail': 'account-verified',
+    'newDeviceLoginEmail': 'new-device-signin',
+    'recoveryEmail': 'forgot-password',
+    'resetEmail': 'forgot-password',
+    'suspiciousLocationEmail': 'suspicious-location',
+    'verifyEmail': 'welcome',
+    'verifyLoginEmail': 'new-signin',
+    'verificationReminderFirstEmail': 'hello-again',
+    'verificationReminderSecondEmail': 'still-there',
+    'verificationReminderEmail': 'hello-again'
```

Comment:
This is a little confusing, [verificationReminerEmail](https://github.com/mozilla/fxa-auth-mailer/blob/master/mailer.js#L451) is actually the mailer function that gets called, and `verificationReminderFirstEmail`, `verificationReminderSecondEmail` are the templates names which are determined by the `type` param. This is the only mailer function that can send different email types depending on a param.
 
Would it make more sense to put `verificationReminerEmail` testing in its own section, to clear up some of the confusion? I agree that each email template have its own distinct campaign.

@vladikoff Thoughts?